client/src/document/index.tsx

```diff
@@ -20,6 +20,7 @@ import { BrowserCompatibilityTable } from "./ingredients/browser-compatibility-t
 import { Breadcrumbs } from "../ui/molecules/breadcrumbs";
 import { Titlebar } from "../ui/molecules/titlebar";
 import { TOC } from "../ui/molecules/toc";
+import { RenderSideBar } from "../ui/organisms/sidebar";
```

Comment:
This is still kinda unpleasant in my view. I really think we should stop and re-assess the use of `ui`. A sidebar only makes sense within a document, but now it has to go up and above one level and then drop deep into another level. 

As mentioned before, things specifically for *MDN documents* should sit in `src/document/**`. General UI things that could be found on any web app can belong on `src/ui/**`. Also, I could see how there can be other utilities that you might employ such as `import { rememberCollapsables } from "../utils"`.

I know I know. It's pickery but it's nice to have the discussion fully and get these things right since there's very little turning back on this. And also, sorry for being the inertia of rapid progress. 

If the reason we're doing these kinds of non-namespaced cross imports is for the benefit of Storyboard, then I'd like to contest whether Storyboard is right for us here and now. Is it that critical to the multi-year future to have every component in Storyboard so it can be viewed in a separate app? What's wrong with just using a document to preview the sidebar? Yes, I know the kitchensink isn't great because it can only have 1 sidebar at a time. (something we could fix by the way!)